src/entrypoints/my-index.ts

```diff
@@ -32,6 +34,13 @@ class MyIndex extends LitElement {
 
   public connectedCallback() {
     super.connectedCallback();
+    if (isMobile && changeRequestedFromRedirect) {
+      const parts = decodeURIComponent(changeRequestedFromRedirect).split("?");
+      const params = new URLSearchParams(parts[1]);
+      params.append("mobile", "1");
+      const url = `/redirect/${parts[0]}?${params.toString()}`;
```

Comment:
Ensure you put a `/` at the end of the url or else Netlify does a 301 redirect.

```suggestion
      const url = `/redirect/${parts[0]}/?${params.toString()}`;
```